packages/astro/src/@types/astro.ts

```diff
@@ -1375,6 +1375,72 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		devOverlay?: boolean;
+
+		// TODO review with docs team before merging to `main`
+		/**
+		 * @docs
+		 * @name experimental.i18n
+		 * @type {object}
+		 * @version 3.*.*
+		 * @type {object}
+		 * @description
+		 *
+		 * Allows to configure the beaviour of the i18n routing
+		 */
+		i18n?: {
+			/**
+			 * @docs
+			 * @name experimental.i18n.defaultLocale
+			 * @type {string}
+			 * @version 3.*.*
+			 * @description
+			 *
+			 * The default locale of your website/application
```

Comment:
> This should probably also give guidance like what form/system you expect, even with an external link, so that people know what syntax to use here.
> 

We haven't mentioned this in the RFC, it would be great if you could add a comment there. We haven't set any restrictions, what do you suggest?

> I'd also indicate whether this is required.

Do I need to use a special JSDoc tag? Or should we just say it in the description?